src/org/ohdsi/usagi/MappingTarget.java

```diff
@@ -72,4 +74,14 @@ public Type getMappingType() {
 	public void setMappingType(Type mappingType) {
 		this.mappingType = mappingType;
 	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
Similarly, hashCode is needed. For objects, you can use `Objects.hash(concept, mappingType)`